packages/notification-center/src/hooks/useNotifications.test.tsx

```diff
@@ -14,7 +14,9 @@ import { IUserPreferenceSettings } from '@novu/client';
 import { ISession, INotificationsContext } from '../shared/interfaces';
 import { NovuProvider } from '../components';
 import { useNotifications } from './useNotifications';
+import { queryClient } from '../components/novu-provider/NovuProvider';
 
+const PROMISE_TIMEOUT = 150;
```

Comment:
I don't know why but increasing the timeout helps tests to pass :D

Reply:
I wonder how hard was figuring it out 😅